src/pages/studyView/tabs/SummaryTab.tsx

```diff
@@ -196,6 +196,10 @@ export class StudySummaryTab extends React.Component<IStudySummaryTabProps, {}>
                 props.getData = () => this.store.getMutatedGenesDownloadData();
                 props.genePanelCache = this.store.genePanelCache;
                 props.downloadTypes = ["Data"];
+                props.filterByCancerGenes= this.store.filterMutatedGenesTableByCancerGenes;
+                props.onToggleCancerGeneFilter = (filtered: boolean) => {
+                    this.store._filterMutatedGenesTableByCancerGenes = filtered;
```

Comment:
if the property has an underscore, i think that is supposed to indicate it's private.  but here we mutate it from outside.    i realize we have consistent convention here.     also, this should be runInAction.